src/main/java/online/book/store/controller/BookController.java

```diff
@@ -0,0 +1,35 @@
+package online.book.store.controller;
+
+import java.util.List;
+import lombok.AllArgsConstructor;
+import online.book.store.dto.BookDto;
+import online.book.store.dto.CreateBookRequestDto;
+import online.book.store.service.BookService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@AllArgsConstructor
+@RestController
+@RequestMapping(value = "/api/books")
```

Comment:
add `server.servlet.context-path=/api` to application.properties

```suggestion
@RequestMapping("/books")
```